tests/build/_internal/test_cmake_file.py

```diff
@@ -2,13 +2,13 @@
 # Copyright (C) 2020 Arm Mbed. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
-from unittest import TestCase
+import pytest
```

Comment:
We shouldn't need to import `pytest` as we're not using anything from that module in commit  "tests: Refactor CMake config test to use pytest". The pytest runner doesn't require this import statement to pick up the test.

Reply:
Yes, I only need pytest later when I start using a fixture. I'll move it commits.